pyiron/atomistics/structure/atoms.py

```diff
@@ -1096,7 +1096,9 @@ def get_extended_positions(self, width):
             dist (float): Distance in Angstrom
 
         Returns:
-            pyiron.atomistics.structure.atoms.Atoms: The required boundary region
+            pyiron.atomistics.structure.atoms.Atoms, numpy.ndarray:
+                Positions of all atoms in the extended box, indices of atoms in
+                their original option (if return_indices=True)
```

Comment:
Is this function currently used by users or is this intended for internal use? 
Since this changes the default behavior, we could include ehm, a note? (Do we have that concept?) 
The default behavior of this function changed as of ... - To restore the old behavior specify `return_indices=True`

Reply:
Oh in general you're right, but since this function is still extremely new (maybe a month old?) and I never introduced it anywhere, I don't think there's anyone using it.

Reply:
Ok, I just thought I would run a notebook and it just does not work anymore xD
Then I am fine with it!